### PR TITLE
Update documentation for Bertly 2.0!

### PR DESCRIPTION
**Pull request by DFurnes**
### What's this PR do?

This pull request updates Bertly's README and API documentation now that we've launched Bertly 2.0!

### How should this be reviewed?

It might be easier to review these on the branch: [README](https://github.com/DoSomething/bertly/tree/documentation#readme) and [documentation](https://github.com/DoSomething/bertly/tree/documentation/documentation).

Does everything make sense? Could anything be clearer? :eyes: 

### Any background context you want to provide?

:book: 

### Relevant tickets

N/A

### Checklist

- [ ] This PR has been added to the relevant Pivotal card.
- [ ] Added appropriate feature/unit tests.

### CODE REVIEW

File: src/Transformers/LinkTransfomer.js [hunk 1]
```diff
@@ -13,6 +13,6 @@ export const transform = link => ({
   count: link.count,
   url: link.url,
   url_short: new URL(link.key, config('app.url')),
-  updatedAt: link.updatedAt.toISOString(),
-  createdAt: link.createdAt.toISOString(),
+  updated_at: link.updatedAt.toISOString(),
+  created_at: link.createdAt.toISOString(),
```
Comment (DFurnes) [inline src/Transformers/LinkTransfomer.js:17]: These are not used anywhere yet (since they didn't exist in Bertly 1.0). I'm just renaming for consistency.

**Comment by DFurnes** [comment]
Hmm, why didn't this automatically request a review...? :thinking:

**Comment by DFurnes** [comment]
Ah!! I think it's because the applications team only had ["read" privileges](https://github.com/orgs/DoSomething/teams/applications/repositories) on this repo. I swapped to "maintain".

File: README.md [hunk 2]
```diff
@@ -9,64 +9,38 @@ Bertly! :link:
...
-$ aws configure --profile serverless-production
-AWS Access Key ID [None]: **************
-AWS Secret Access Key [None]: **************
-Default region name [None]: us-east-1
-Default output format [None]: text
+npm test
 ```
 
-Then, run either `npm run deploy:dev`, `npm run deploy:qa`, or `npm run deploy:prod` to deploy! (For more power, you can also install the [Serverless CLI](https://serverless.com/framework/docs/getting-started/) globally on your machine and run commands with `serverless` or `sls`).
+Please write a test case when adding or changing a feature. Most steps you would take when manually testing your code can be automated, which makes it easier for yourself & others to review your code and ensures we don't accidentally break something later on!
```
Comment (katiecrane) [inline README.md:33]: YES!

File: documentation/README.md [hunk 3]
```diff
@@ -2,38 +2,33 @@
...
-Given a URL, return a JSON object containing the shortened URL, and a token to use when revoking the shortened URL. If the same URL is provided multiple times, the same short-link will be returned.
+The request body is accepted as either `application/json` or `application/x-www-form-urlencoded`.
 
 <details>
 <summary><strong>Example Request</strong></summary>
 
 ```sh
 curl -X "POST" "https://dosome.click/" \
-     -H 'X-BERTLY-API-KEY: ******' \
-     -H 'Content-Type: application/x-www-form-urlencoded; charset=utf-8' \
```
Comment (katiecrane) [inline documentation/README.md:36]: Does this no longer need to be specified in the request?

File: documentation/README.md [hunk 4]
```diff
@@ -84,18 +80,20 @@ curl "https://dosome.click/wq544/clicks"
...
 ```json
 {
-  "count": 42,
-  "status": "okay",
-  "url": "https://www.github.com/dosomething/bertly"
+  "key": "wq544",
+  "url": "https://www.github.com/dosomething/bertly",
+  "url_short": "https://dosome.click/wq544",
+  "updated_at": "2020-08-03T19:36:43.695Z",
+  "created_at": "2020-06-09T18:13:08.663Z"
```
Comment (katiecrane) [inline documentation/README.md:87]: Shouldn't this include a `count` still?

File: documentation/README.md (hunk 3 above)
Comment (DFurnes) [reply documentation/README.md:36]: Never did! It's automatically set when using the `--data-urlencode` option.

File: documentation/README.md (hunk 4 above)
Comment (DFurnes) [reply documentation/README.md:87]: Oops, it should! Where'd that go?! 🙈

File: documentation/README.md (hunk 3 above)
Comment (katiecrane) [reply documentation/README.md:36]: ah I see that down there now!

**Pull request merged**